luminex/src/org/labkey/luminex/query/NegativeBeadDisplayColumnFactory.java

```diff
@@ -58,60 +61,68 @@ public String getFormFieldName(RenderContext ctx)
             }
 
             @Override
-            public void renderTitle(RenderContext ctx, Writer out) throws IOException
+            public @NotNull HtmlString getTitle(RenderContext ctx)
             {
-                out.write("<script type=\"text/javascript\" nonce=\"" + HttpView.currentPageConfig().getScriptNonce() + "\">\n" +
+                return HtmlStringBuilder.of()

```

Comment:
ewwww

Reply:
@labkey-matthewb you like the new version better?